### PR TITLE
Explain why "docker attach" may appear to hang

### PR DESCRIPTION
**Pull request by gschaetz**
issue #1456 https://github.com/docker/docker.github.io/issues/1456

### CODE REVIEW

**Comment by gschaetz** [comment]
This fixes a doc issue at https://github.com/docker/docker.github.io/issues/2005

File: docs/reference/commandline/attach.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ Use `docker attach` to attach your terminal's standard input, output, and error
 ID or name. This allows you to view its ongoing output or to control it
 interactively, as though the commands were running directly in your terminal.
 
+> **Note:**
+> The attach command will display the output of the ENTRYPOINT/CMD process.  This
```
Comment (mstanleyjones) [inline docs/reference/commandline/attach.md]: Please mark this up a bit by putting technical words like "attach" and "ENTRYPOINT/CMD" in back-ticks.

Only a single space between sentences.

Other than that, this is great!

**Review by mdlinville — CHANGES_REQUESTED** [review]

**Comment by gschaetz** [comment]
@mstanleyjones can you see If I got this right now.

File: docs/reference/commandline/build.md [hunk 2]
```diff
@@ -80,6 +80,10 @@ the command succeeds, the directory is sent to the Docker daemon as the
 context. Local clones give you the ability to access private repositories using
 local user credentials, VPN's, and so forth.
 
+> **Note:**
```
Comment (mstanleyjones) [inline docs/reference/commandline/build.md]: This change is unrelated to `docker attach` and should probably be submitted in a separate PR. I think this time we can let it go, but in future, please try to keep your PRs to one issue at a time, for ease of review and for if the change ever needs to be reverted.

**Comment by mstanleyjones** [comment]
@vdemeester PTAL

**Review by mdlinville — APPROVED** [review]

**Review by thaJeztah — CHANGES_REQUESTED** [review]
changes look good to me, but can you squash your commits, so that there's a single commit in this PR? You can find some instructions in our contributors guide; https://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently

**Comment by mstanleyjones** [comment]
@gschaetz we would like to close this one out, can you please address the feedback? :)

**Comment by gschaetz** [comment]
@mstanleyjones I will get it done tonight

**Review by vdemeester — CHANGES_REQUESTED** [review]
LGTM 👼 but @gschaetz you need to squash the 5 commits into one 🙏

**Comment by gschaetz** [comment]
> @vdemeester I'll try to squash them again. 
>

**Comment by gschaetz** [comment]
@vdemeester  I tried to rebase and squash but the old ones still show up.  Any tips you can give me or instructions?

**Comment by vdemeester** [comment]
@gschaetz sure 😉 

```
$ git rebase --interactive HEAD~5
# keep one and mark the other as squash in the editor that popped up
# update commit message if it's messed up and save it
$ git status 
# to make sure you have one commit and the remote has 5
$ git push --force-with-lease origin master
```

*Note for later: it's safer to open a PR from a separate branch as master (allows you to keep updating master without messing current PR 👼 )*

**Comment by gschaetz** [comment]
Ugh @vdemeester that is pulling in other commits.  Not just mine.

**Review by thaJeztah — APPROVED** [review]
@gschaetz I rebased/squashed and pushed to your branch to update the PR

LGTM

**Pull request merged**